infra/data-pipeline.yml

```diff
@@ -444,6 +444,42 @@ tasks:
         owner: release-mgmt-analysis@mozilla.com
         source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
 
+    - ID: train-similarity
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '118 hours'}
+      expires: {$fromNow: '1 month'}
+      provisionerId: aws-provisioner-v1
+      workerType: relman-svc-compute
+      dependencies:
+        - bugs-retrieval
+      payload:
+        maxRunTime: 25200
+        image: mozilla/bugbug-base:${version}
+        command:
+          - bugbug-train-similarity
+          - bm25
```

Comment:
The command is "bugbug-train-similarity bm25", shouldn't it be "bugbug-train-similarity --algorithm bm25"?

Reply:
@marco-c I'm trying to figure out if I should I add `- --algorithm` or `- algorithm`?

Reply:
It's just like the shell, so "--" before algorithm.